package proto_ff;

import "yd_fieldoptions.proto";


message E_DailyDaily
{
	optional int64 m_id = 1[(yd_fieldoptions.field_cname) = "类型ID"];
	optional string m_name = 2[(yd_fieldoptions.field_cname) = "显示名称", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 m_type = 3[(yd_fieldoptions.field_cname) = "活动类型"];
	optional int32 m_args = 4[(yd_fieldoptions.field_cname) = "玩法参数"];
	optional int32 m_playtype = 5[(yd_fieldoptions.field_cname) = "玩法类型"];
	optional int32 m_playargs = 6[(yd_fieldoptions.field_cname) = "玩法参数"];
	optional int64 m_funciontid = 7[(yd_fieldoptions.field_cname) = "功能ID"];
	optional int32 m_number = 8[(yd_fieldoptions.field_cname) = "次数"];
	optional int32 m_viptype = 9[(yd_fieldoptions.field_cname) = "VIP类型"];
	optional int32 m_othernum = 10[(yd_fieldoptions.field_cname) = "其余可购买次数"];
	optional int32 m_eachscore = 11[(yd_fieldoptions.field_cname) = "每次活跃值"];
	optional int32 m_cycle = 12[(yd_fieldoptions.field_cname) = "重置周期"];
	optional int32 m_regreward = 13[(yd_fieldoptions.field_cname) = "报名奖励道具boxid"];
}


message Sheet_DailyDaily
{
	repeated E_DailyDaily E_DailyDaily_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message E_DailyReward
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "索引"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 m_activescore = 3[(yd_fieldoptions.field_cname) = "活跃度值"];
	optional int64 m_rewardid = 4[(yd_fieldoptions.field_cname) = "奖励ID"];
	optional int32 m_rewardnum = 5[(yd_fieldoptions.field_cname) = "奖励数量"];
}


message Sheet_DailyReward
{
	repeated E_DailyReward E_DailyReward_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message E_DailyCultivateAttributeDesc
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_value = 2[(yd_fieldoptions.field_cname) = "值"];
}

message E_DailyCultivate
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_lv = 2[(yd_fieldoptions.field_cname) = "等级"];
	optional int32 m_exp = 3[(yd_fieldoptions.field_cname) = "升级所需经验"];
	repeated E_DailyCultivateAttributeDesc m_attribute = 4[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 4];
}


message Sheet_DailyCultivate
{
	repeated E_DailyCultivate E_DailyCultivate_List = 1[(yd_fieldoptions.field_arysize)=40];
}
